flexible_event_config.md

```diff
@@ -29,10 +32,10 @@ This proposal is broken into two separate feature sets:
   * This lite version provides a subset of the full feature and can be used independently of Phase 2.
   * Implemented [as described in the primary explainer](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#optional-varying-frequency-and-number-of-reports).
 * __Phase 2: Full version of flexible event-level configuration__
-
-Phase 2 (Full flexible event-level) could be used to do all of the capabilities in Phase 1 and:
-* Vary the trigger data cardinality in a report
-* Reduce the amount of total noise by decreasing the trigger data cardinality
+  * This full version provides all of the capabilities in Phase 1 and:
```

Comment:
I'm still a bit confused by the word `full` here, since Phase 1 + custom trigger data is not the full set of capabilities proposed by this document. Specifically, there is also the concept of summary buckets.